certsuite/report/report.css

```diff
@@ -20,6 +20,7 @@ body {
 }
 .PASS { background-color: green }
 .FAIL { background-color: red }
+.OK { background-color: lightgreen}
```

Comment:
This 'OK' is the current tag in code, it only shows in cert test for parent test. I suggest let it be OK now and adjust it to right tag after considering all tags to wrap to new tag name.